src/drivers/intel/haswell/timer.c

```diff
@@ -115,10 +115,10 @@ void timer_unregister(struct timer *timer)
 
 void timer_enable(struct timer *timer)
 {
-	interrupt_enable(timer->irq);
+	interrupt_enable(timer->irq, 0);
```

Comment:
re-reverts some previous patches.

Reply:
Sorry, I cannot find that in my patches? Maybe an even earlier patch removed that parameter?